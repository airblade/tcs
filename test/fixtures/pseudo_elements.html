<span class="after:content-['*'] after:ml-0.5 after:text-red-500 block text-sm font-medium text-slate-700">
  Email
</span>

<span class="first-line:uppercase"></span>
