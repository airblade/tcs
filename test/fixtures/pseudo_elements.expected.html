<span class="block text-sm font-medium text-slate-700 after:ml-0.5 after:text-red-500 after:content-['*']">
  Email
</span>

<span class="first-line:uppercase"></span>
